src/20240118_StkGHO_Activation/StkGHOActivation.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "GHO Safety Module Initialization"
+author: "Aave_BGDLabs_ACI"
```

Comment:
Similar to other proposals we review, we are not to be included here. Usually who implements the proposal and who creates it are included (Aave Lab & ACI).